### PR TITLE
Static grader

### PR DESCRIPTION
**Pull request by chrislerus**

### CODE REVIEW

**Comment by tianhuil** [comment]
@chrislerus: can you setup testing for this repository?

File: static_grader/grader_server.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from scorer import SCORERS_DICT
+from scorers import DICT_SCORERS
```
Comment (cmoscardi) [inline static_grader/grader_server.py]: Is there a reason this isn't a relative import `.scorers`?

File: static_grader/scorers.py [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+"""
+
+import os
+import sys
+import re
+
+import numpy as np
+import requests
+
+from serializable import SerializedScore
```
Comment (cmoscardi) [inline static_grader/scorers.py]: Similarly `.serializable` here?

**Comment by tianhuil** [comment]
Two small things: can you add this to Jenkins and put this under pylint?  You'll probably have to write a custom `.pylintrc` file and set tabs to two spaces (and maybe other rules).

**Pull request merged**